webrtc.html

```diff
@@ -1910,8 +1908,8 @@
 
         <p>An <code><a>RTCPeerConnection</a></code> object MUST not be garbage
         collected as long as any event can cause an event handler to be
-        triggered on the object. When the object's
-        <a>signaling state</a> is <code>closed</code>, no such event handler can be
+        triggered on the object. When the object's [[<a>isClosed</a>]]
+        internal slot is <code>true</code>, no such event handler can be
```

Comment:
I interpret that section as containing additional rules that come into play when no JS references are left. Otherwise it's not OK to attempt to gc in the first place.